packages/core/src/js/feedback/FeedbackFormScreen.tsx

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * Implements a feedback form screen that sends feedback to Sentry using Sentry.captureFeedback.
+ */
+export class FeedbackFormScreen extends React.Component<FeedbackFormScreenProps, FeedbackFormScreenState> {
+  public constructor(props: FeedbackFormScreenProps) {
+    super(props);
+    this.state = {
+      name: '',
+      email: '',
+      description: '',
+    };
+  }
+
+  public handleFeedbackSubmit: () => void = () => {
+    const { name, email, description } = this.state;
+    const { closeScreen, text } = this.props;
+
+    const trimmedName = name?.trim();
+    const trimmedEmail = email?.trim();
+    const trimmedDescription = description?.trim();
+
+    if (!trimmedName || !trimmedEmail || !trimmedDescription) {
+      const errorMessage = text?.formError || 'Please fill out all required fields.';
+      Alert.alert(text?.errorTitle || 'Error', errorMessage);
+      return;
+    }
+
+    if (!this._isValidEmail(trimmedEmail)) {
+      const errorMessage = text?.emailError || 'Please enter a valid email address.';
+      Alert.alert(text?.errorTitle || 'Error', errorMessage);
+      return;
+    }
+
+    const userFeedback: SendFeedbackParams = {
+      message: trimmedDescription,
+      name: trimmedName,
+      email: trimmedEmail,
+    };
+
+    captureFeedback(userFeedback);
+    closeScreen();
+  };
+
+  /**
+   * Renders the feedback form screen.
+   */
+  public render(): React.ReactNode {
+    const { closeScreen, text, styles } = this.props;
+    const { name, email, description } = this.state;
+
+    return (
+    <View style={styles?.container || defaultStyles.container}>
+      <Text style={styles?.title || defaultStyles.title}>{text?.formTitle || 'Feedback Form'}</Text>
+
+      <TextInput
+        style={styles?.input || defaultStyles.input}
+        placeholder={text?.namePlaceholder || 'Name'}
+        value={name}
+        onChangeText={(value) => this.setState({ name: value })}
+        />
+
+      <TextInput
+        style={styles?.input || defaultStyles.input}
+        placeholder={text?.emailPlaceholder || 'Email'}
+        keyboardType={'email-address' as KeyboardTypeOptions}
+        value={email}
+        onChangeText={(value) => this.setState({ email: value })}
+        />
+
+      <TextInput
+        style={[styles?.input || defaultStyles.input, styles?.textArea || defaultStyles.textArea]}
+        placeholder={text?.descriptionPlaceholder || 'Description (required)'}
+        value={description}
+        onChangeText={(value) => this.setState({ description: value })}
+        multiline
+      />
+
+      <TouchableOpacity style={styles?.submitButton || defaultStyles.submitButton} onPress={this.handleFeedbackSubmit}>
+        <Text style={styles?.submitText || defaultStyles.submitText}>{text?.submitButton || 'Send Feedback'}</Text>
+      </TouchableOpacity>
+
+      <TouchableOpacity style={styles?.cancelButton || defaultStyles.cancelButton} onPress={closeScreen}>
```

Comment:
I think for close we could guide users to add custom `onFormClose`.

Similar to success but with guide.

```
This shows in development only.
Use `onFormClose` to navigate back
or hide the form component.
By default the form will hide in release builds.
<Button>Reopen form<Button>
```

To avoid any mistakes in Release build the form by default should just unmount.

Reply:
Updated to use `onFormClose` and added unmounting as the default behavior with 501a134fea6f6188abf4672e2aa8a67e640b05b1